geode-redis/README.md

```diff
@@ -25,7 +25,7 @@ and then connect the [Redis-CLI](https://redis.io/topics/quickstart) to that ins
 
 ### <a name="building-apache-geode"></a>Supported Redis Commands
 
-Not all Redis commands are currently supported. The current set of supported Redis commands is listed [here](#redis-command-status).
+Not all Redis commands are currently supported. The current set of supported Redis commands is listed [here](#supported-redis-commands).
 
```

Comment:
Please do not say "currently", as it constitutes a promise for future updates that makes the product sound half-baked and unstable. Just say "Not all Redis commands are supported."

Reply:
removed.